sotodlib/tod_ops/sub_polyf.py

```diff
@@ -1,8 +1,176 @@
 import numpy as np
 import logging
+from scipy.special import eval_legendre
 from . import flags
 logger = logging.getLogger(__name__)
 
+def _mask_data(_data, mask):
+    data = np.ma.masked_array(_data,mask=mask)
+    return data
+
+def subscan_legendre_filter(aman, degree, signal=None, exclude_turnarounds=False, mask=None, in_place=True):
```

Comment:
Definition of `degree` is inconsistent with `subscan_polyfilter`.
Now, `degree in subscan_polyfilter` = `degree in subscan_legendre_filter + 1`.
Maybe you can refer the `max_mode` definition in azss:
https://github.com/simonsobs/sotodlib/blob/0ca1d61e7e7b73869b3afb71363c9a7b0c8f01a8/sotodlib/tod_ops/azss.py#L59-L60

Reply:
That's correct.
Thanks.